packages/bun-uws/src/HttpResponse.h

```diff
@@ -581,6 +581,12 @@ struct HttpResponse : public AsyncSocket<SSL> {
         return this;
     }
 
+    HttpResponse* clearOnAborted() {
+        HttpResponseData<SSL> *httpResponseData = getHttpResponseData();
+
+        httpResponseData->onAborted = nullptr;
```

Comment:
```suggestion
        httpResponseData->onAborted = nullptr;
        httpResponseData->onWritable = nullptr;        
```

Reply:
fixed in https://github.com/oven-sh/bun/pull/12472/commits/6094340309d0fddecd377aeabb706d04d516afed we clean both only in some cases (onWritable being set or not changes some internal behaviors)